Sources/test/PBISequence.fcl

```diff
@@ -0,0 +1,28 @@
+#include "fcl/minimalMessageService.fcl"
+process_name: PBISequenceTest
+
+services: {
+  message : @local::default_message
+}
+
+source: {
+  module_type :  PBISequence 
+  fileNames: ["/pnfs/mu2e/persistent/users/brownd/SS_2020-05-23.dat"]
+  runNumber : 1
+}
+
+Output : {
+  module_type : RootOutput
+  outputCommands:   [ "drop *_*_*_*", "keep mu2e::ProtonBunchIntensity_*_*_*" ]
```

Comment:
Please extend the keep command so that you keep the Subrun summary object.  For example:

  outputCommands:   [ "drop *_*_*_*",
                      "keep mu2e::ProtonBunchIntensity_*_*_*",
                      "keep mu2e::ProtonBunchIntensitySummary_*_*_*" ]